Scheme.md

```diff
@@ -0,0 +1,89 @@
+When loading an `isolated-app:` URL, the user agent checks the signatures of the underlying web bundle and whether it trusts the public key that was used to sign it.
+A user agent might trust, for example, public keys configured via enterprise policy, public keys of known distribution mechanisms/stores, or separately allow-listed public keys of individual Isolated Web Apps.
+`isolated-app:` URLs are considered Secure Contexts, just like HTTPS pages, and thus have access to APIs like Service Workers. `isolated-app:` URLs look like so:
+
+```
+isolated-app://signed-web-bundle-id/path/inside/app.js?some-query#foo
+^           ://^                   /^                 ?^         #^
+scheme         opaque host          path               query      fragment
+```
+
+### Host
+
+URLs with the `isolated-app:` scheme use a Signed Web Bundle ID (see next section) as their opaque host, as defined in the [URL standard](https://url.spec.whatwg.org/).
+The Signed Web Bundle ID being the host adds the requirement that it must be a valid hostname (see [RFC 3986](https://www.rfc-editor.org/rfc/rfc3986)), which is the case given that Signed Web Bundle IDs are base32 encoded without padding.
+It is the user agent’s responsibility to map the Signed Web Bundle ID contained in the opaque host to an installed Isolated Web App and then use the URL path to serve resources from its Web Bundle.
+
+### Port & Credentials
+
+Isolated Web App URLs do not use usernames, passwords, or ports.
+All three of these components are therefore always `null`.
+Isolated Web App URLs with a port or credentials should not be loaded and result in an HTTP error.
+
+### Path, Query, Fragment
+
+These work like one would expect from HTTP(S) URLs, as described in the URL specification.
+
+
+## Signed Web Bundle IDs
+
+We identify Isolated Web Apps by their _Signed Web Bundle ID_.
+Signed Web Bundle IDs consist of an identifier followed by a suffix that represents the type of identifier.
+Both parts are concatenated and base32-encoded, discarding any potential padding.
+The result is then transformed into lowercase.
+The suffix itself consists of two parts: The last byte indicates how many bytes are used to describe the identifier type, and the `n` bytes before it are used to describe the identifier type.
+The general format of Signed Web Bundle IDs looks like this:
+
+```
+lowercase(base32EncodeWithoutPadding([identifier] [identifier type] [identifier type length]))
+```
+
+Currently, the following two suffixes are defined:
+
+`0x00 0x00 0x02`: This suffix indicates that the ID is a user-agent-specific value that can be used for testing and development purposes.
+For example, loading a Web Bundle from the local filesystem which is not signed.
+
+`0x00 0x01 0x02`: This suffix is used for Ed25519 public keys as Signed Web Bundle ID.
```

Comment:
```suggestion
`0x00 0x01 0x02`: This suffix is used for [Ed25519](https://tools.ietf.org/id/draft-ietf-curdle-ssh-ed25519-02.xml) public keys as Signed Web Bundle ID.
```

Reply:
I added a link to https://datatracker.ietf.org/doc/html/rfc8032 instead, which I think is a better reference.